src/mcts/node.cc

```diff
@@ -235,7 +235,7 @@ void Node::MakeNotTerminal() {
   n_ = 0;
 
   // If we have edges, we've been extended (1 visit), so include children too.
-  if (edges_) {
+  if (child_) {
```

Comment:
I don't really understand a change in this line, could you comment?
(also maybe the comment above this line needs change?)

Reply:
Modified the comment to clarify why child_ and the behavior for when there's no child_.